src/flow-util.h

```diff
@@ -34,10 +34,6 @@
         (f)->tosrcpktcnt = 0; \
         (f)->todstbytecnt = 0; \
         (f)->tosrcbytecnt = 0; \
-        (f)->todstbypasspktcnt = 0; \
```

Comment:
these were introduced in this PR too, can this and https://github.com/OISF/suricata/pull/3895/commits/51e46ed27f1a1d0653c477889e2aed044430fc61 be ordered differently so we don't make this flip-flop part of the git history?